layouts/partials/docs/native-libraries.md

```diff
@@ -0,0 +1,31 @@
+<!-- prettier-ignore -->
```

Comment:
I merged the java "Libraries" page into "Instrumentation ecosystem". I wanted to reference the a portion of the "docs/languages/libraries-info.md" shortcode to generate a list of native instrumentation from the registry, but the introduction to library instrumentation included in the shortcode didn't make sense in the context of the page.

So I split the generation of the native instrumentation library list into a partial, which is called by the "docs/languages/libraries-info.md" shortcode. The java "Instrumentation ecosystem" page calls the new partial (via the "docs/languages/native-libraries.md" pass-through without the introduction text. 

WDYT?

Reply:
Should be ok, though I think that the argument handling needs to be looked at carefully (to ensure that the shortcode usage in other pages doesn't look a bit wonky now). I'll take a look as soon as I can. @svrnm said that he'd also take a look if/when he has the time.

Reply:
I pulled the PR and looked at other pages that use the shortcode and they look fine 👍

Reply:
Marking this thread resolved.

Reply:
Thanks for double checking @svrnm.